apps/server/src/modules/learnroom/service/metadata-loader.service.spec.ts

```diff
@@ -3,7 +3,7 @@ import { MikroORM } from '@mikro-orm/core';
 import { NotImplementedException } from '@nestjs/common';
 import { Test, TestingModule } from '@nestjs/testing';
 import { LearnroomTypes } from '@shared/domain';
-import { CourseRepo } from '@shared/repo';
+import { CourseRepo, LessonRepo } from '@shared/repo';
 import { courseFactory, setupEntities } from '@shared/testing';
 import { MetadataLoader } from './metadata-loader.service';
 
```

Comment:
I think now that the metadata-loader is no longer used in the shareservice, it can be deleted.

Reply:
metadata-loader deleted